packages/block-library/src/post-terms/block.json

```diff
@@ -11,6 +11,10 @@
 		},
 		"textAlign": {
 			"type": "string"
+		},
+		"separator": {
+			"type": "string",
+			"default": ", "
```

Comment:
Should default be empty?

Reply:
I think a comma makes more sense — it tends to be the most commonly used separator in themes.